renovate/best-practice-all-managers.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  "extends": [
+    "github>konflux-ci/mintmaker//config/renovate/renovate.json",
+    "config:best-practices"
+  ],
+  "enabledManagers": [
+    "tekton",
+    "dockerfile",
+    "rpm",
+    "regex",
```

Comment:
let's fix it to `custom.regex` to avoid generate pr like this https://github.com/app-sre/github-mirror/pull/162

Reply:
Fixed